product-tours/tour-tf-keras/Tour-With-TensorFlow-and-Keras.ipynb

```diff
@@ -0,0 +1,2217 @@
+      "c0563b7a2e8948fd9978bc9354ca6065",
+      "decaf1c07cae4c77b93cda67fe099e84",
+      "c0e4c494d7f1477ca12efba2d5d4fa0d",
+      "66caa712c02441ee828d621a3e71de54",
+      "23cdc7027a414473aa7c2d0f9ca87fa8",
+      "2da128bac24f45f8ab91a14f07b277ec",
+      "4ad8f9ca40b9434c809bb8320eaf880e",
+      "0273d66512434d10bbb9a11401e83972",
+      "12a5f6c96bc3440da1190b534247193e",
+      "3d4296d82a534741be3416d621259e69",
+      "93ecc7bd7d194062a2b10b5d6498f323",
+      "ce82c5ef3840466e84bc4f76e6b15a5b",
+      "7f657e4fe7414f579d1a0fcf72030869",
+      "ecf95e257ab3486abe8d6e4b6c61f118",
+      "6a94ff719378494989cff5ab48c3a5cf",
+      "d920dd3a4fd24bc0a542f2f61b9833a2",
+      "fe01222bf018498a899d97eb8a992400",
+      "b822595237fb402d9a6adae103a6a5cd",
+      "fbe67705da6e4009b16de22718d14b4a",
+      "b280c8ec9fde4027aa6160ccabdd1168",
+      "509ab2788f8345fca1ea48d2069d0ab6",
+      "11083e4a55114a9e88196a53f26c8684",
+      "74e02a36f2a54e7ebd3b3b00274d0fce",
+      "0f44e8ac4bc646428843cd047a7072ce",
+      "b950ad13f0ca4d80bb3dca375f7442a0",
+      "84beace795f2438fb559fc8f4e1ed506",
+      "54f93b1aa7404964ae93a32bfae2a9ff",
+      "b56e9e4063cf48dc9e67c004ace5b8ce",
+      "9e8bba26d9574ebb9b9591c46bf4a1b1",
+      "bdceecbff7024109b6a6a1e11723107a",
+      "8eea965837be4b468bbb852e18d8dde6",
+      "822b579efc5f4e6f90e9d1f025334f7a",
+      "9c9c27cb7dba44e0a9425ccec5dc05d0",
+      "6c22d754f0c04ab683e33ccde7b34f82",
+      "59915949271044d68f7df41436f03625",
+      "3512f48f7b674367a93b346b67431a38",
+      "85aca891429d4943be2d8dd02cf342b3",
+      "b239f7cc9624462e889a329eb763cb07"
+     ]
+    },
+    "colab_type": "code",
+    "id": "3u_Kb86BzWzq",
+    "outputId": "120f001e-e748-4fa1-8405-8e5d3c3dfd16",
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "fig, ax = plt.subplots()\n",
+    "plot_roc(y_test, y_pred_proba, ax=ax)\n",
+    "neptune.log_image('model-performance-visualizations', fig, image_name='ROC')\n",
+    "\n",
+    "fig, ax = plt.subplots()\n",
+    "plot_precision_recall(y_test, y_pred_proba, ax=ax)\n",
+    "neptune.log_image('model-performance-visualizations', fig, image_name='precision recall')\n",
+    "plt.close('all')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "tests"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# tests\n",
+    "exp = neptune.get_experiment()\n",
+    "\n",
+    "## check logs\n",
+    "correct_logs = ['train data sample', 'model_summary', 'batch_loss', 'batch_accuracy',\n",
+    "               'epoch_loss', 'epoch_accuracy', 'test_loss', 'test_accuracy',\n",
+    "                'epoch_val_loss', 'epoch_val_accuracy',\n",
+    "                'model-performance-visualizations']\n",
+    "\n",
+    "if set(exp.get_logs().keys()) != set(correct_logs):\n",
+    "    raise ValueError('incorrect metrics')\n",
+    "\n",
+    "neptune.stop()"
```

Comment:
This should not be in the test but after -> the splitting in the CI will remove it. Perhaps here it doesn't matter but with more than one experiments in a script it will fail CI

Reply:
changed.